rdke-raspberrypi.xml

```diff
@@ -3,6 +3,7 @@
     <remote fetch="ssh://git@github.com/rdk-e" name="rdke" review="https://github.com/rdk-e"/>
     <remote fetch="https://github.com/rdkcentral" name="rdkcentral" review="https://github.com/rdkcentral"/>
     <remote fetch="https://code.rdkcentral.com/r/a/" name="cmf" review="https://code.rdkcentral.com/r/a/"/>
+    <remote fetch="https://git.yoctoproject.org" name="vendor" review="https://git.yoctoproject.org"/>
```

Comment:
Can we change `name="vendor"` to `name="yocto"`?
So that it is clear where it is pulling from.